packages/z/zbar/xmake.lua

```diff
@@ -0,0 +1,57 @@
+package("zbar")
+    set_homepage("https://github.com/mchehab/zbar")
+    set_description("Library for reading bar codes from various sources")
+    set_license("LGPL-2.1")
+
+    add_urls("https://github.com/mchehab/zbar/archive/refs/tags/$(version).tar.gz",
+             "https://github.com/mchehab/zbar.git")
+    add_versions("0.23.93", "212dfab527894b8bcbcc7cd1d43d63f5604a07473d31a5f02889e372614ebe28")
+
+    if is_plat("mingw") and is_subhost("msys") then
+        add_extsources("pacman::zbar")
+    elseif is_plat("linux") then
+        add_extsources("pacman::zbar", "apt::libzbar-dev")
+    elseif is_plat("macosx") then
+        add_extsources("brew::zbar")
+    end
+
+    if is_plat("linux", "bsd") then
+        add_syslinks("pthread")
+    end
+
+    if is_plat("macosx") then
+        add_deps("libiconv", {system = true})
```

Comment:
libiconv will find and use system libs first. so you need not add `system = true`